src/Navigation.ts

```diff
@@ -1,22 +1,59 @@
 import { activePlugins } from './Utils/constants';
+import { HrefNavItem, NavSection } from '@console/dynamic-plugin-sdk/src';
 
-export default async ({ dynamicNav, currentNamespace }) => {
+export interface RouteProps {
+  isHidden?: boolean;
+  appId?: string;
+  href?: string;
+  title?: string;
+}
```

Comment:
Oh, FWIW... interface is a hold-over type for objects (`interface` was the only way in early TS)... recommend just using `type` now. But it really doesn't matter either way.

Eg:
```suggestion
export type RouteProps = {
  isHidden?: boolean;
  appId?: string;
  href?: string;
  title?: string;
};
```